lib/matplotlib/style/core.py

```diff
@@ -83,6 +83,8 @@ def use(style):
             try:
                 rc = rc_params_from_file(style, use_default_template=False)
                 mpl.rcParams.update(rc)
+            except URLError:
+                raise
             except IOError:
                 msg = ("'%s' not found in the style library and input is "
                        "not a valid URL or path. See `style.available` for "
```

Comment:
Sure, I just mean that as you filter out the URLError part of the IOError above, then this existing error message for IOError now suggests reasons that no longer apply to this error (because they have already been filtered out).